src/pages/tech/MalfunctionsTable.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import { makeStyles, Theme, createStyles, Paper, Table, TableBody, TableCell, TableContainer, TableHead, TableRow } from '@material-ui/core';
+import { Malfunction } from '../../models/malfunctions';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    content: {
+      flexGrow: 1,
+      padding: theme.spacing(3),      
+      display: "flex",
+      width: "100%",
+      justify: "center", 
+      alignItems: "center",      
+    },
+    table: {
+      minWidth: 700,
+    },
+    blockButton: {
+      color: "#ee6002"
+    },
+    unblockButton: {
+        color: "#09af00"
+    },
+    addButton: {
+      margin: theme.spacing(2),
+    },
+  })
+);
+
+export interface MalfunctionsTableProps {
+  setMalfunctions: (value: React.SetStateAction<Malfunction[]>) => void;  
+  malfunctions: Malfunction[];
+}
+
+const MalfunctionsTable = (props: MalfunctionsTableProps) => {
+    const classes = useStyles()
+   
+  return (
+    <TableContainer component={Paper}>
+    <Table className={classes.table} aria-label="simple table">
+      <TableHead>
+        <TableRow>
+          <TableCell>Bike</TableCell>
+          <TableCell align="right">User Id</TableCell>
```

Comment:
Can all the headers be displayed in one line? "User Id" and "Bike Id" are wrapped.

Reply:
good now